projects/plugins/boost/wp-js-data-sync.php

```diff
@@ -162,5 +165,47 @@ function jetpack_boost_initialize_datasync() {
 $css_excludes_entry = new Minify_Excludes_State_Entry( 'minify_css_excludes' );
 jetpack_boost_register_option( 'minify_js_excludes', Schema::as_array( Schema::as_string() )->fallback( Minify_JS::$default_excludes ), $js_excludes_entry );
 jetpack_boost_register_option( 'minify_css_excludes', Schema::as_array( Schema::as_string() )->fallback( Minify_CSS::$default_excludes ), $css_excludes_entry );
+jetpack_boost_register_option(
+	'image_cdn_quality',
+	Schema::as_assoc_array(
+		array(
+			'jpg'  => Schema::as_assoc_array(
+				array(
+					'quality'  => Schema::as_number(),
+					'lossless' => Schema::as_boolean()->fallback( false ),
```

Comment:
If the lossless fallback is false, there should also probably be a number fallback to the original quality?